molgenis-data-cache/src/main/java/org/molgenis/data/cache/l2/L2Cache.java

```diff
@@ -115,11 +114,7 @@ public Entity get(Repository<Entity> repository, Object id)
 		catch (ExecutionException exception)
 		{
 			// rethrow unchecked
-			if (exception.getCause() != null && exception.getCause() instanceof RuntimeException)
-			{
-				throw (RuntimeException) exception.getCause();
-			}
-			throw new MolgenisDataException(exception);
+			throw new RuntimeException(exception);
```

Comment:
Why this change?

Reply:
MolgenisDataException is deprecated, it's not something to show to the user, so RuntimeException would be fine I think